docusaurus/docs/reactnative/basics/translations.mdx

```diff
@@ -107,11 +246,22 @@ Use the default English language date-times instead of those dictated by the lan
 ### language
 
 Language code for language to be used.
+The following options are available:
```

Comment:
Borrowing a convention from REST docs. For fields like this that have an enumerated list of valid values, it's helpful to include a reference.